src/App.tsx

```diff
@@ -1,25 +1,9 @@
 import * as React from 'react';
-import {createBrowserRouter, RouterProvider} from 'react-router-dom';
-import TeamOverview from './pages/TeamOverview';
-import Teams from './pages/Teams';
-import UserOverview from './pages/UserOverview';
+import {RouterProvider} from 'react-router-dom';
+import Router from './Router';
 
 const App = () => {
-    var router = createBrowserRouter([
-        {
-            path: '/',
-            element: <Teams />,
-        },
-        {
-            path: '/team/:teamId',
-            element: <TeamOverview />,
-        },
-        {
-            path: '/user/:useId',
-            element: <UserOverview />,
-        },
-    ]);
-    return <RouterProvider router={router} />;
```

Comment:
created router as a component, in the app file we can do more things like analytics, redux integration etc.

Reply:
this will make code more readable and extendable.